lib/Models/Catalog/Ows/WebMapServiceCapabilitiesStratum.ts

```diff
@@ -718,16 +719,33 @@ export default class WebMapServiceCapabilitiesStratum extends LoadableStratum(
 
   @computed
   get supportsGetLegendGraphic(): boolean {
+    const capabilities = this.capabilities?.json?.Capability;
+
     return (
       isDefined(this.capabilities?.json?.["xmlns:sld"]) ||
-      isDefined(
-        this.capabilities?.json?.Capability?.Request?.GetLegendGraphic
-      ) ||
+      isDefined(capabilities?.Request?.GetLegendGraphic) ||
```

Comment:
Sure, just running some more check for getLegendGraphic support, as that support might be indicated in other places, especially for ncWMS? Or does this fix legend support for other WMS too?

Reply:
Yeah it just adds support for ncWMS servers that advertise `GetLegendGraphic` in the `<ExtendedCapabilities><ExtendedRequest>` section - as defined by the ncWMS spec

For example here

https://github.com/TerriaJS/terriajs/blob/66938054c85893dfc4683a253f2df513021be241/wwwroot/test/WMS/colorscalerange.xml#L133-L148

Whereas GeoServer will advertise support in `<Request>` section - as defined by the WMS-SLD spec

For example

https://github.com/TerriaJS/terriajs/blob/c002e67e6c906fc83b653ffc8f63d6843288212a/wwwroot/test/WMS/single_metadata_url.xml#L68-L78

I believe ncWMS `GetLegendGraphics`just  add additional properties on top of WMS-SLD `GetLegendGraphics`.

I did some testing across multiple THREDDs and noticed no issues

Reply:
OK looks good!